velox/exec/HashJoinBridge.cpp

```diff
@@ -198,4 +191,9 @@ uint64_t HashJoinMemoryReclaimer::reclaim(
       });
   return reclaimedBytes;
 }
+
+bool isHashBuildMemoryPool(const memory::MemoryPool& pool) {
```

Comment:
Is this a test-only function? If so, would you move it into test code?

Reply:
It is used in production code as well. But it is also used in test in this PR.

Reply:
I didn't notice it is used in HashJoinMemoryReclaimer::reclaim. Feels a bit flaky to check the name.

Reply:
Yes, I agree. But both hash probe and join listed under the same plan node. But it won't be a problem if we always partition hash build input per our offline discussion. Then all these logic can be simplified.